examples/plot_rgb.py

```diff
@@ -0,0 +1,168 @@
+landsat_bands_data_path = "data/vignette-landsat/LC08_L1TP_034032_20160621_20170221_01_T1_sr_band*[1-7]*.tif"
+stack_band_paths = glob(landsat_bands_data_path)
+stack_band_paths.sort()
+
+# Create image stack and apply nodata value for Landsat
+arr_st, meta = es.stack(stack_band_paths, nodata=-9999)
+
+###############################################################################
+# Plot RGB Composite Image
+# --------------------------
+# You can use ``plot_rgb()`` function from the ``earthpy.plot`` module to quickly
+# plot three band composite images. For RGB composite images, you will plot the red,
+# green, and blue bands, which are bands 4, 3, and 2, respectively, in the image
+# stack you created. Python uses a zero-based index system, so you need to subtract
+# a value of 1 from each index. Thus, the index for the red band is 3, green is 2,
+# and blue is 1. These index values are provided to the ``rgb`` argument to identify
+# the bands for the composite image.
+
+# Create figure with one plot
+fig, ax = plt.subplots(figsize=(12, 12))
+
+# Plot red, green, and blue bands, respectively
+ep.plot_rgb(arr_st, rgb=(3, 2, 1), ax=ax, title="Landsat 8 RGB Image")
+plt.show()
+
+###############################################################################
+# Stretch Composite Images
+# -------------------------
+# Composite images can sometimes be dark if the pixel brightness values are
+# skewed toward the value of zero. You can stretch the pixel brightness values
+# in an image using the argument ``stretch=True`` to extend the values to the
+# full 0-255 range of potential values to increase the visual contrast of the
+# image. In addition, the ``str_clip`` argument allows you to specify how much of
+# the tails of the data that you want to clip off. The larger the number, the
+# more the data will be stretched or brightened.
+
+# Create figure with one plot
+fig, ax = plt.subplots(figsize=(12, 12))
+
+# Plot bands with stretched applied
+ep.plot_rgb(
+    arr_st,
+    rgb=(3, 2, 1),
+    ax=ax,
+    stretch=True,
+    str_clip=0.5,
+    title="Landsat 8 RGB Image with Stretch Applied",
+)
+plt.show()
+
+###############################################################################
+# Plot Color Infrared (CIR) Composite Image
+# ------------------------------------------
+# For color infrared (CIR) composite images, you will plot the near-infrared (NIR),
+# red, and green bands, which are bands 5, 4, 2, respectively. Once again, the
+# Python index values will be the original band number minus 1, thus, 4, 3, and 2
+# for NIR, red, and green, respectively.
+
+# Create figure with one plot
+fig, ax = plt.subplots(figsize=(12, 12))
+
+# Plot NIR, red, and green bands, respectively, with stretch
+ep.plot_rgb(
+    arr_st,
+    rgb=(4, 3, 2),
+    ax=ax,
+    stretch=True,
+    str_clip=0.5,
+    title="Landsat 8 CIR Image with Stretch Applied",
+)
+plt.show()
+
+###############################################################################
```

Comment:
please look at the build of this. When i build i see a very large white space at the top and bottom of the figure. if you set the figsize() to ensure the height is SMALLER THAN the width it will fix the white space issue. 12,6 seemed to be decent for the size on my machine locally.

<img width="773" alt="Screen Shot 2019-05-24 at 9 21 18 AM" src="https://user-images.githubusercontent.com/7649194/58338643-5c4d0b00-7e05-11e9-9c39-bfbd7fee992c.png">

Reply:
Fixed!